.babelrcAlt

```diff
@@ -0,0 +1,19 @@
+{
+  "presets": [
+    [
+      "@babel/env",
+      {
+        "modules": false,
+        "useBuiltIns": false,
+        "loose": true,
+        "targets": "chrome > 100"
+      }
+    ],
+    [
+      "@babel/typescript",
+      {
+        "allowDeclareFields": true
+      }
+    ]
+  ]
+}
```

Comment:
this would be an alternative config file, very loose, that would behave as tsc, just stripping types.
This could be useful for developers that want to import files directly and then want to bundle and transpile on their own.